src/sentry/analytics/events/comment_webhooks.py

```diff
@@ -0,0 +1,30 @@
+import abc
+
+from sentry import analytics
+
+
+class CommentEvent(analytics.Event, abc.ABC):
+    attributes = (
+        analytics.Attribute("user_id", type=int, required=False),
+        analytics.Attribute("group_id", type=int),
+        analytics.Attribute("project_id", type=int),
```

Comment:
In the analytics Event you're taking a `project_id` but in the `analytics.record()`, you're passing in a `project_slug`.

Reply:
woops